content/pages/veteran-id-card/how-to-get.md

```diff
@@ -1,124 +0,0 @@
----
```

Comment:
Removed content that was referenced in the ignore list. We should specify content availability in the targets.